### PR TITLE
Fix MPS import in TFQ

### LINKED ISSUE #393: mps is not included in BUILD.

### PR DESCRIPTION
**Pull request by jaeyoo**
Resolves #393

### CODE REVIEW

File: lib/BUILD [hunk 1]
```diff
@@ -125,6 +129,8 @@ cc_library(
         "io.h",
         "io_file.h",
         "matrix.h",
+        "mps_simulator.h",
+        "mps_statespace.h",
```
Comment (95-martin-orion) [inline lib/BUILD:133]: qsimh (the hybrid Schrodinger-Feynman simulator) is largely unused at this point. Is it necessary to include these here?

**Review by 95-martin-orion — CHANGES_REQUESTED** [review]
Mostly seems fine, one comment on qsimh.

File: lib/BUILD (hunk 1 above)
Comment (jaeyoo) [reply lib/BUILD:133]: Oh my mistake. let me remove it then :) Thank you @95-martin-orion

**Review by 95-martin-orion — APPROVED** [review]
Looks good to me. To confirm, #393 is resolved by this, but not #395 - is that correct?

**Comment by jaeyoo** [comment]
> Looks good to me. To confirm, #393 is resolved by this, but not #395 - is that correct?

Yes, right. #394 resolves #393 . #396 resolves #395 . Thanks!

**Comment by sergeisakov** [comment]
> Mostly seems fine, one comment on qsimh.

I think the same comment is valid for "run_qsim_lib". This should have been added only to "qsim_lib".

**Pull request merged**